src/core/storage/query_engine/operators/lambda_transform.hpp

```diff
@@ -189,8 +189,7 @@ class operator_impl<planner_node_type::LAMBDA_TRANSFORM_NODE> : public query_ope
       res.soft_assign(val);
       return res;
     } else {
-      std::string message = "Cannot convert " + std::string(val) +
-          " to " + flex_type_enum_to_name(type);
+      std::string message = "Cannot convert " + std::string(val) + " of type " + flex_type_enum_to_name(val.get_type()) + " to " + flex_type_enum_to_name(type) +  " which is the type of the SArray. If this is not the correct type try specifying it during creation";
```

Comment:
This line is really long. It would be nice to break it into a few shorter lines.